civihr_default_theme/scss/base/_task-assignments-modal-temp.scss

```diff
@@ -14,10 +14,14 @@ $white: #FFFFFF;
     overflow-y: scroll !important;
 
     .modal-dialog {
-      width: 500px;
+      max-width: 500px;
```

Comment:
Add details about both these changes, why they need to be specified and what is the affect on UI with screenshot before and After.